### PR TITLE
Create hello.js

### PR DESCRIPTION
**Pull request by ChristoBose**

### CODE REVIEW

File: hello.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+let x = Myfunction(4, 3);
```
Comment (ashna-53) [inline hello.js]: The function name should be in camelcase. Please change it.

File: hello.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+function Myfunction(A, B) {
```
Comment (ashna-53) [inline hello.js]: Change function name to camecase

File: hello.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+// Function returns the product of a and b
+  return A * B;
```
Comment (ashna-53) [inline hello.js]: Change the variable name to small letter

File: hello.js (hunk 1 above)
Comment (ChristoBose) [reply hello.js]: changes are done.please check

File: hello.js (hunk 2 above)
Comment (ChristoBose) [reply hello.js]: changes are done.please check

File: hello.js (hunk 3 above)
Comment (ChristoBose) [reply hello.js]: changes are done please check

**Pull request merged**